model/DataQuery.php

```diff
@@ -779,6 +779,22 @@ public function selectFromTable($table, $fields) {
 	}
 
 	/**
+	 * Add the given fields from the given table to the select statement.
+	 *
+	 * @param String $table Unquoted table name (will be escaped automatically)
+	 * @param Array $fields Database column names (will be escaped automatically)
+	 */
+	public function addSelectFromTable($table, $fields) {
+		$fieldExpressions = array_map(function($item) use($table) {
+			return "\"$table\".\"$item\"";
```

Comment:
Ah, sorry - I probably thought this was against master. If we're connecting in ansi mode then `quoteSymbol` should quote in ansi mode otherwise it wouldn't be doing a very good job!